decidim-conferences/app/cells/decidim/conferences/conference_cell.rb

```diff
@@ -6,13 +6,19 @@ module Conferences
     # the default size is the Medium Card (:m)
     class ConferenceCell < Decidim::ViewModel
       def show
-        cell card_size, model
+        cell card_size, model, options
```

Comment:
Is this options parameter referring to  ?
https://github.com/trailblazer/cells/blob/7f620b408a965d89805e88791d6528605f758b15/lib/cell/view_model.rb#L139

Reply:
Yes, almost all equivalent cells defined in the application use options in the same way. In the case of this cell the options were not being used, but now they are because a highlight parameter can be included.